libs/init/migrate/api/src/mill/init/migrate/Tree.scala

```diff
@@ -0,0 +1,24 @@
+package mill.init.migrate
+
+import upickle.default.{ReadWriter, macroRW}
+
+case class Tree[+A](root: A, children: Seq[Tree[A]] = Nil) {
```

Comment:
How is this different from `libs/init/buildgen/src/mill/main/buildgen/Tree.scala`?